apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -34,9 +34,14 @@ const DescriptionEditor = ({ isEditable }: { isEditable: boolean }) => {
     setIsMounted(true);
   }, []);
 
+  function convertLineBreaksToHTML(inputText: string): string {
+    if (!inputText || inputText === "") return;
+    return inputText.replace(/\n/g, "<br>");
+  }
+
```

Comment:
Nice work @SySagar , Can you turn this into a library or a hook? This component will be useful later, so instead of redeclaring it, we can simply import it.

Reply:
oh sure, will do that asap. Just one doubt Since there were a lot of commits and basically one task so I have squashed them into one single commit. Is that ok?

Reply:
Also is there a common global hooks directory where I can put or is there any other way?

Reply:
Declare it inside `packages/lib` , then import and use